lighthouse-cli/chrome-finder.ts

```diff
@@ -75,14 +75,53 @@ export function darwin() {
 }
 
 export function linux() {
+  const installations: Array<string> = [];
+  const suffixes = [
+    '/share/applications/chromium-devel.desktop',
+    '/share/applications/google-chrome.desktop'
+  ];
+  const installationFolders = [
+    process.env.HOME + '/.local/',
+    '/usr/',
+  ];
+  installationFolders.forEach(folder => {
+    suffixes.forEach(suffix => {
+      const desktopPath = path.join(folder, suffix);
+
+      if (desktopPath && canAccess(desktopPath)) {
```

Comment:
Wouldn't `desktopPath` always be a truthy value?
https://nodejs.org/api/path.html#path_path_join_paths